src/_utils/itineraryLocation/index.tsx

```diff
@@ -0,0 +1,86 @@
+import styled from 'styled-components'
+import { color, space, componentSizes } from '_utils/branding'
+
+import ItineraryLocation from './ItineraryLocation'
+
+const StyledItineraryLocation = styled(ItineraryLocation)`
+  & {
+    --time-width: calc(${componentSizes.timeWidth} + ${componentSizes.bulletSize});
+  }
+
+  & {
+    position: relative;
+    list-style-type: none;
+  }
+
+  /* wrapper */
+  & .kirk-itineraryLocation-wrapper {
+    display: flex;
+    padding: ${space.m} 0;
+    width: 100%;
+  }
+
+  & a.kirk-itineraryLocation-wrapper {
+    background: none;
+    text-decoration: none;
+    user-select: none;
+    -webkit-tap-highlight-color: ${color.tapHighlight};
+    -webkit-touch-callout: none;
+  }
+
+  & button.kirk-itineraryLocation-wrapper {
+    border: 0;
+    cursor: pointer;
+    text-align: left;
+    width: 100%;
+    font-family: inherit;
+    outline: none;
+    background-color: transparent;
+    -webkit-tap-highlight-color: ${color.tapHighlight};
+    -webkit-touch-callout: none;
+  }
+
+  & a.kirk-itineraryLocation-wrapper:hover,
+  & button.kirk-itineraryLocation-wrapper:hover {
+    background-color: ${color.tapHighlight};
+  }
+
+  /* city */
+  & time {
+    min-width: var(--time-width);
+  }
+
+  & .kirk-itineraryLocation-chevron {
+    display: flex;
```

Comment:
I would add `align-self: center;` so the icon appears at the middle of the location item